plugins/plugin-chart-echarts/package.json

```diff
@@ -31,7 +31,8 @@
     "@types/echarts": "^4.6.3",
     "@types/mathjs": "^6.0.7",
     "echarts": "^5.0.0",
-    "mathjs": "^8.0.1"
+    "mathjs": "^8.0.1",
+    "moment": "^2.29.1"
```

Comment:
Please `import { getTimeFormatter } from '@superset-ui/core';`.

We use d3 time format for all other charts, if we want to add support for `moment` format, we should do it centrally in `superset-ui`, not in each chart.